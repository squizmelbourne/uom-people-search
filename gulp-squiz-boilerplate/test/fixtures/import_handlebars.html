<!-- import:handlebars test/fixtures/templates/template.hbs {"data": "test/fixtures/templates/data.json", "helpers": "test/fixtures/js/helpers.js"} -->