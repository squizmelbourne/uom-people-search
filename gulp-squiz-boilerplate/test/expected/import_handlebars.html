<ul>
    <li>1: one</li>
    <li>2: two</li>
    <li>3: three</li>
</ul>